src/ol/object.js

```diff
@@ -163,6 +163,8 @@ ol.ObjectAccessor.prototype.transform = function(from, to) {
  * details, and see {@link ol.dom.Input} for the specific case of binding an
  * object with an HTML element.
  *
+ * Own observable properties may be removed by calling `unset()`.
```

Comment:
The description of the class writes about `Classes that inherit from this
have pre-defined properties, to which you can add your own.`

You are right `own observable properties` is a bit too much.
What about using `own properties` to differenciate from the `pre-defined` properties?